Documentation/Testing.md

```diff
@@ -110,6 +110,23 @@ git checkout my-css-change
 ./Meta/WPT.sh run --log results.log 
 ```
 
+### Importing Web Platform Tests
+
+You can use the `Meta/import-wpt-test.py` script to import Web Platform Tests (WPT) tests into your Ladybird clone (and you _should_ — especially if any code changes you’re making cause Ladybird to pass any WPT tests it hasn’t yet been passing). Here’s how:
```

Comment:
> (and you _should_ — especially if any code changes you’re making cause Ladybird to pass any WPT tests it hasn’t yet been passing)

It might be worth de-emphasizing this point slightly, as there are **a lot** of tests that can't be imported in this way at the moment (reftests, anything that uses networking features, etc).

Reply:
> It might be worth de-emphasizing this point slightly, as there are **a lot** of tests that can't be imported in this way at the moment (reftests, anything that uses networking features, etc).

Thanks, changed the wording around that — please take a look